indico/modules/events/papers/operations.py

```diff
@@ -0,0 +1,29 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask import session
+
+from indico.modules.events.papers import logger
+from indico.modules.events.logs.models.entries import EventLogRealm, EventLogKind
+
+
+def toggle_reviewing_type(event, reviewing_type, enable):
+    event.cfp.toggle_reviewing(reviewing_type, enable)
+    logger.info("Reviewing type '%s' for event %r toggled (%s) by %r", reviewing_type, event, enable, session.user)
```

Comment:
I think it would be cleaner to show `enabled`/`disabled` like you are doing in `event.log`.